application/models/Campaign_model.php

```diff
@@ -1847,7 +1847,14 @@ public function validation_check($id, $title, $url, $format = null)
 
         $tmp_dir = $this->config->item('archive_dir');
 
-        $header = curl_header($url, false, $tmp_dir);
+        $header = null;
+        try {
+            $header = curl_header($url, false, $tmp_dir);
+        }
+        catch (Exception $e) {
+            return false;
```

Comment:
Probably doesn't matter, but I think this is the one place we want to throw the exception. The method is only called from a `full-scan` and none of the callers look at the return value.

Idk, were you seeing any other test failures before adding the exception handling here?

Reply:
Nope. I'm going to hold off on throwing the exception because I think it won't get caught before getting dumped to the output and I'd rather keep whatever error-handling behavior we have now.